frontend/src/components/widget/Lookup/RawLookup.js

```diff
@@ -271,6 +271,7 @@ export class RawLookup extends Component {
           if (!mandatory) {
             this.fireOnDropdownListToggle(true);
           }
+          this.handleInputTextChange(undefined, true);
```

Comment:
what's the reason of calling this?

Reply:
to call `handleInputTextChange` with `allowEmpty` set to true, to ensures that clicking on the input field triggers the same logic as typing a space.